.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1979,33 +2121,47 @@ iam_Scenario_CreateUserAssumeRole:
             - description: Create functions that wrap &IAM; user actions.
               snippet_tags:
                 - iam.java2.scenario.main
-    Python:
+    JavaScript:
       versions:
         - sdk_version: 3
-          github: python/example_code/iam/iam_basics
+          github: javascriptv3/example_code/iam
           sdkguide:
           excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - iam.JavaScript.scenario.createclientv3
             - description: Create an &IAM; user and a role that grants permission
                 to list &S3; buckets. The user has rights only to assume the role.
                 After assuming the role, use temporary credentials to list buckets for
                 the account.
               snippet_tags:
-                - python.example_code.iam.Scenario_CreateUserAssumeRole
-    JavaScript:
+                - javascript.iam_scenarios.iam_basics
+    PHP:
       versions:
         - sdk_version: 3
-          github: javascriptv3/example_code/iam
+          github: php/example_code/iam/iam_basics
           sdkguide:
           excerpts:
-            - description: Create the client.
+            - description: Shows how to use &AWS; SDK for PHP v3 to get started using &IAMlong;
+                (&IAM;). Create a user that has no permissions. Create a role and policy that grant s3:ListAllMyBuckets
+                permission. Grant the user permission to assume the role. Create an &S3; client object as the user and try
+                to list buckets, which will fail due to permission restrictions. Get temporary credentials by assuming
+                the role. Create an &S3; client object with the temporary credentials and list the buckets (this will now
+                succeed). Delete all the resources.
```

Comment:
As a suggestion, this block looks mostly redundant to the `synopsis_list` above. They render one after the other in the output, so maybe remove this description block (and, yes, the other examples should probably follow this as well)?

Reply:
removed